collections/Ash_Rolls/MS_Ash_Rolls_38.xml

```diff
@@ -94,24 +94,21 @@
                <additional>
                   <adminInfo>
                      <recordHist>
-                        <source>Description adapted (2018) from Watson and Pächt and Alexander (see bibliography), incorporating additional description of decoration by Elizabeth Solopova, c. 2000. Previously described in the Quarto Catalogue (W. H. Black, <title>A descriptive, analytical, and critical catalogue of the manuscripts bequeathed unto the University of Oxford by Elias Ashmole Esq....</title>, Quarto Catalogues X, 1845). <listBibl>
-                              <bibl facs="aeh0010.gif" type="QUARTO">Quarto Catalogue X, col. 10 (no. 23)</bibl>
-                              <bibl facs="aaq0486.gif" type="SC">Summary Catalogue, vol. 2, part 2, p. 1127</bibl>
-                           </listBibl>
-                        </source>
+                        <source>Description adapted (2018) from Watson and Pächt and Alexander (see bibliography), incorporating additional description of decoration by Elizabeth Solopova, c. 2000.</source>
                      </recordHist>
                   </adminInfo>
                   <surrogates><bibl subtype="partial" type="digital-facsimile"><ref target="https://digital.bodleian.ox.ac.uk/objects/ff06fc08-3038-429b-9d3d-f913160b972b/"><title>Digital Bodleian</title></ref>
                         <note>(2 images from 35mm slides)</note></bibl></surrogates>
-                  <listBibl type="WRAPPER">
-                     <listBibl>
-                        <bibl>A. G. Watson, <title>Catalogue of Dated and Datable Manuscripts c.435–1600 in Oxford Libraries</title> (Oxford, 1984), no. 38</bibl>
-                        <bibl>Otto Pächt and J. J. G. Alexander, <title>Illuminated Manuscripts in the Bodleian Library Oxford</title>, III (1973), no. 535</bibl>
-                     </listBibl>
-                     <listBibl type="INTERNET">
-                        <head>Online resources:</head>
-                        <bibl><ref target="http://jonas.irht.cnrs.fr/manuscrit/39784"><title>JONAS: Répertoire des textes et des manuscrits médiévaux d'oc et d'oïl</title></ref></bibl>
-                     </listBibl>
+                  <listBibl>
```

Comment:
I've sorted this alphabetically, but a different approach would be by date, as the British Library does with their online catalogue entries.